packages/appstore/src/modules/traders-hub/account-with-wallets.tsx

```diff
@@ -1,115 +1,31 @@
-            loginid: 'MFW12345',
-        },
-        {
-            balance: 0,
-            currency: 'AUD',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 3476.21,
-            currency: 'USD',
-            landing_company_shortcode: 'malta',
-            is_virtual: 0,
-            loginid: 'MFW12345',
-        },
-        {
-            balance: 0,
-            currency: 'ETH',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 2.45,
-            currency: 'BTC',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 0,
-            currency: 'LTC',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 0,
-            currency: 'USDC',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 0,
-            currency: 'USDT',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 0,
-            currency: 'eUSDT',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-        {
-            balance: 0,
-            currency: 'tUSDT',
-            landing_company_shortcode: 'svg',
-            is_virtual: 0,
-            loginid: 'CRW12345',
-        },
-    ];
-
-    // TODO: delete 'wallet_test_accounts' after testing
-    const wallet_accounts = show_test_wallets
-        ? wallet_test_accounts
-        : Object.keys(accounts)
-              .filter(key => accounts[key]?.account_category === 'wallet')
-              .reduce((acc, cur) => {
-                  acc.push({ ...accounts[cur], loginid: cur });
-                  return acc;
-              }, [] as typeof accounts[0][]);
+    if (error) return <p>{JSON.stringify(error)}</p>;
 
     return (
         <React.Fragment>
-            {wallet_accounts.map((account, index) => (
-                <Wallet
-                    key={`${account.account_type} ${account.landing_company_shortcode} ${account.currency} ${index}`}
-                    wallet_account={account}
-                />
-            ))}
+            {data?.map(wallet => {
+                return (
+                    <Wallet
+                        key={wallet.loginid}
+                        wallet_account={wallet}
+                        active={selected_wallet === wallet.loginid}
+                        setActive={() =>
+                            setSelectedWallet(prevWallet => (prevWallet === wallet.loginid ? '' : wallet.loginid || ''))
```

Comment:
```suggestion
                            setSelectedWallet(previous => (previous === wallet.loginid ? undefined : wallet.loginid))
```
@thisyahlen-deriv If you want to toggle better to using undefined instead of empty string to indicate nothing is selected.